cmd/terminate.go

```diff
@@ -42,6 +43,8 @@ func init() {
 	terminateCmd.Flags().StringSliceVarP(&instanceIdFlag, "instance-ids", "n", nil,
 		"The instance ids of the instances you want to terminate")
 	terminateCmd.Flags().BoolVarP(&isInteractive, "interactive", "i", false, "Interactive mode")
+	terminateCmd.Flags().StringToStringVar(&flagConfig.UserTags, "tags", nil,
+		"Terminate instances containing EXACT tag key-pair (Example: CreatedBy=simple-ec2)")
```

Comment:
looks like `=` in the tag key will mess up the format expected by `StringToStringVar`. simple-ec2 should probably restrict `=` in tag key? 
Ref: [StringToString](https://sourcegraph.com/github.com/spf13/pflag/-/blob/string_to_string.go#L26)

Reply:
Did some local testing, and I don't think we need to explicitly restrict `=` in tag key. With that said, **it is not possible** to have `=` in the tag key with this implementation (your ref shows key will always be left of the first `=`).

Confirmed this behavior: whenever I put multiple `=` or try to escape it, the additional `=` will be prepended to the value.

Ex:
![image](https://user-images.githubusercontent.com/61433408/106815435-73fc0f80-6639-11eb-9e5d-43772f4e1e49.png)


![image](https://user-images.githubusercontent.com/61433408/106815464-8118fe80-6639-11eb-95fa-df8015f5eebd.png)

Reply:
Sure. I was thinking of a tag like `--tags keyPart1=keyPart2=Value` --> resultant key = `keyPart1`, value = `keyPart2=Value`. Just wanted to bring this up as `=` is a valid char. I don't really have a real-world use case in mind. Could probably be addressed at a later time as this seems like an edge case.

Reply:
ahh gotcha-- yeah we can address it later. When I do the PR to update ReadMe with all the new features I'll be sure to include a callout